network.go

```diff
@@ -400,3 +415,55 @@ func (c *Client) GetDeletedNetworks(ctx context.Context) ([]Network, error) {
 	}
 	return networks, err
 }
+
+// GetPinnedServerList get a list of pinned servers in a specific network.
```

Comment:
I would assume that the GoLangDoc line should be a complete sentence. 
`GetPinnedServerList returns a list of pinned servers in a specific network.`